plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -135,6 +135,10 @@ public String getUsername()
         // enable case sensitive (see the CS below) collation for SQL identifiers
         container.addEnv("MSSQL_COLLATION", "Latin1_General_CS_AS");
 
+        // TLS and certificate validation are on by default, and need
+        // to be disabled for tests.
+        container.withUrlParam("encrypt", "false");
```

Comment:
Also need to pass `trustServerCertificate=false`.

From the release notes of driver:

> | Certificate validation when encrypt = false | BREAKING CHANGE When  encrypt = false but the server requires encryption, the certificate will  be validated based on the trustServerCertificate connection setting. |
> |:-|:-|